pkg/occlient/occlient.go

```diff
@@ -1951,10 +1997,10 @@ func (c *Client) GetOneServiceFromSelector(selector string) (*corev1.Service, er
 	return &services[0], nil
 }
 
-// AddEnvironmentVariablesToDeploymentConfig adds the given environment
+// UpdateEnvironmentVariablesToDeploymentConfig adds the given environment
 // variables to the only container in the Deployment Config and updates in the
 // cluster
-func (c *Client) AddEnvironmentVariablesToDeploymentConfig(envs []corev1.EnvVar, dc *appsv1.DeploymentConfig) error {
+func (c *Client) UpdateEnvironmentVariablesToDeploymentConfig(envs []corev1.EnvVar, dc *appsv1.DeploymentConfig) error {
```

Comment:
Why was this function renamed?
Using 'Upade' makes it look like env variables in DC are replaced instead of added to the existing one.